doc/overview.rst

```diff
@@ -2394,15 +2448,15 @@ configuration file.
 -  Model: ``CreateModel``, ``UpdateModel``
 -  Model Package: ``CreateModelPackage``, ``UpdateModelPackage``
 -  Processing Jobs: ``CreateProcessingJob``, ``UpdateProcessingJob``
--  Training Job: ``CreateTrainingJob``, ``UpdateTrainingJob``
--  Edge Packaging Job: ``CreateEdgePackagingJob``, ``UpdateEdgePackagingJob``
```

Comment:
Same for this one. @akrishna1995 Why do you think we should remove this one?

Reply:
Apologies - I might have not been very clear - 

* We want to remove **UpdateEdgePackagingJob** as there is no API as the same - "https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_UpdateEdgePackagingJob.html" 
* My bad - I removed UpdateTrainingJob because we do not support resource_config injection, but we updated it to include the profiler_config and can add it back